### PR TITLE
Install and setup Chromium

### PR DESCRIPTION
**Pull request by agger-magenta**
If Google Chrome is required, some minimal changes are required to the scripts.

But this is the basic idea.

### CODE REVIEW

File: admin_scripts/os2display_os/chromium_setup.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Make Chromium  autostart and set it up with OS2Display.
+
+# Initialize parameters
+
+TIME=$1
```
Comment (dals83) [inline admin_scripts/os2display_os/chromium_setup.sh]: Why is TIME needed?

Comment (agger-magenta) [reply admin_scripts/os2display_os/chromium_setup.sh]: Because AAK wants a delay before the browser is launched to allow the network to get up.

**Comment by agger-magenta** [comment]
Since this PR is approved, I'll merge it as soon as I've been able to test it with an OS Display system.

**Comment by agger-magenta** [comment]
Closing since these changes are moved to a new `server_image` repository.

**Pull request closed without merging**